src/apps/backend/modules/task/rest-api/task-router.ts

```diff
@@ -1,18 +1,17 @@
-import { Router } from 'express';
+import { accessAuthMiddleware } from '../../access-token';
+import { ApplicationRouter } from '../../application';
 
-import AccountAuthMiddleware from '../../access-token/rest-api/account-auth-middleware';
+import * as TaskController from './task-controller';
 
-import TaskController from './task-controller';
+export default class TaskRouter extends ApplicationRouter {
+  configure(): void {
+    const { router } = this;
 
-export default class TaskRouter {
-  public static getRoutes(): Router {
-    const router = Router({ mergeParams: true });
+    router.use(accessAuthMiddleware);
 
-    router.post('/', AccountAuthMiddleware.ensureAccess, TaskController.createTask);
-    router.get('/', AccountAuthMiddleware.ensureAccess, TaskController.getAllTasks);
-    router.get('/:id', AccountAuthMiddleware.ensureAccess, TaskController.getTask);
-    router.delete('/:id', AccountAuthMiddleware.ensureAccess, TaskController.deleteTask);
-
-    return router;
+    router.post('/', TaskController.createTask);
+    router.get('/', TaskController.getTasks);
+    router.get('/:id', TaskController.getTask);
+    router.delete('/:id', TaskController.deleteTask);
```

Comment:
I have noticed at times that if you give `/abc/:abcid` as a common prefix in the `*-server.ts` file and then append something in the `*-router.ts` file it doesn't quite work.

Reply:
Can you provide some more details on this? I don't have much detail here on the issue in order to address it here.